lib/index.js

```diff
@@ -282,7 +283,9 @@ module.exports = {
     if (
       !isProduction &&
       PRE_BUILT_TARGETS.every((target) => targets.includes(target)) &&
-      targets.length === PRE_BUILT_TARGETS.length
+      targets.length === PRE_BUILT_TARGETS.length &&
+      // if node is defined in targets we can't reliably use the prebuilt bundles
+      !targetNode
```

Comment:
I think this is only true if we are _also_ using `ember-cli-fastboot` or `fastboot`, we can further narrow this down.

Reply:
so... I think you are probably right, in that it wouldn't make much sense in having `node: "current"` or any sort of node target in your targets file if you are not using ember-cli-fastboot or fastboot, but I think the code as written is "more correct".

Essentially we are pre-building the ember files for certain targets, and if our **actual** targets are outside that set then we need to revert to providing the tree rather than the pre-built bundle. It just so happens that we don't target node in our pre-built bundles and this is an instant opt out because of that.

Another think we could explore is to just hash the full targets (not just the browsers) in some way and compare that to the hash of the targets that the pre-built files were built with. This would have the same effect since it will likely never have node in the full targets file.

Thoughts?

Reply:
So we discussed this in the Fastboot meeting and have agreed that it is probably fine to go ahead with. I have confirmed that the concern @rwjblue had about the potential performance impact of this change is not actually an issue. I mentioned in the meeting that it seemed the same speed to build, but that was without any data so I did a quick comparison: 

Linked to the version from this PR: 

```
➜  ember-website git:(master) time ember build

Missing symlinked npm packages:
Package: ember-source
  * Specified: ~3.24.0
  * Symlinked: 3.27.0

Environment: development
cleaning up...
Built project successfully. Stored in "dist/".
ember build  24.98s user 7.98s system 112% cpu 29.295 total
```

Using the latest release of ember (at the time): 

```
time ember build
Environment: development
cleaning up...
Built project successfully. Stored in "dist/".
ember build  25.77s user 7.59s system 122% cpu 27.223 total
```

Those build times are within the margin of error 👍